src/Tools/dotnet-watch/test/GlobbingAppTests.cs

```diff
@@ -21,7 +22,7 @@ public GlobbingAppTests(ITestOutputHelper logger)
             _app = new GlobbingApp(logger);
         }
 
-        [ConditionalTheory]
+        [ConditionalTheory(Skip = "https://github.com/aspnet/AspNetCore/issues/8267")]
```

Comment:
Minor but `[Theory(Skip = "https://github.com/aspnet/AspNetCore/issues/8267")]` or `[Fact(Skip = "https://github.com/aspnet/AspNetCore/issues/8267")]` and removing `[SkipOnHelix]` is more straightforward if we stick w/ skipping these tests.